lib/src/clojure_lsp/db.clj

```diff
@@ -19,7 +19,7 @@
 (defonce created-watched-files-chan (async/chan 1))
 (defonce edits-chan (async/chan 1))
 
-(def version 1)
+(def version 2)
```

Comment:
Very important change here. Since there are already projects with external analysis cache on filesystem, those analysis doesn't have the new :external, so we invalidate that cache here, we could add a `(or (:external? %) true)` all over the project, but it seems ugly, this change should not be a problem.